raven/processes/wps_ostrich_gr4j_cemaneige.py

```diff
@@ -44,6 +44,15 @@
     CEMANEIGE_X2=0.1,
 )
 
+params = LiteralInput(
+    "params",
+    "Comma separated list of model parameters",
+    abstract="UParameters: " + ", ".join(f.name for f in fields(params_defaults)),
+    data_type="string",
+    default=", ".join(map(str, astuple(params_defaults))),
+    min_occurs=0,
+)
+
```

Comment:
Why is this needed ?

Reply:
Difficult to determine from my side. As far as I can tell, my changes don't touch these processes, but still the tests were failing (see my comment above).

Reply:
@cjauvin any idea why the ravenpy update would require params to be defined in ostrich ?

Reply:
I'm not entirely sure but I think it's due to that fact that some emulators (`GR4JCN` in particular) do not override the `derived_parameters` method for their Ostrich counterpart (i.e. `GR4JCN_OST`) which means that the one from parent class is called (`GR4JCN.derived_parameters`), and `self.config.rvp.params` is expected to be defined in it:

https://github.com/CSHS-CWRA/RavenPy/blob/8c5f9ba7792f53821ca96fc0415d72b217180764/ravenpy/models/emulators/hbvec.py#L236-L257

Reply:
But I'm not sure I understand why derived_parameters was removed in 
https://github.com/CSHS-CWRA/RavenPy/commit/59cc90e895a21233d1ff16abe0b000fdbf9cd58a#diff-9fe4fb4dc82db51bc5391ff9a00afacc03509f2c3a1bdf1b0619b41d7946b8ed

Should I just add it back, or is there a new framework I should work with ?